contract_manager/src/chains.ts

```diff
@@ -231,16 +211,25 @@ export class SuiChain extends Chain {
       BigInt(fee),
       BigInt(exponent)
     ).encode();
-    return this.wrapWithWormholeGovernancePayload(3, setFee);
```

Comment:
Since now the implementation is the same as the parent class, you don't need to override them anymore and can just delete the function.